client/state/marketplace/selectors.ts

```diff
@@ -67,4 +70,16 @@ export const hasOrIntendsToBuyLiveSupport = ( state: IAppState ): boolean => {
 	return hasLiveSupport;
 };
 
-export default shouldUpgradeCheck;
+export function canPublishProductReviews(
+	state: IAppState,
+	productType: string,
+	productSlug: string
+) {
+	if ( productType === 'theme' ) {
+		return canPublishThemeReview( state, productSlug );
+	}
+	if ( productType === 'plugin' ) {
+		throw new Error( `Selector nnt implemented for plugins` );
```

Comment:
Nitpick: there is typo in error message.
```suggestion
		throw new Error( `Selector not implemented for plugins` );
```